app/src/main/java/org/mozilla/focus/widget/RoundedImageView.kt

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.mozilla.focus.widget
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.PorterDuff
+import android.graphics.PorterDuffXfermode
+import android.graphics.RectF
+import android.graphics.drawable.BitmapDrawable
+import android.support.v7.widget.AppCompatImageView
+import android.util.AttributeSet
+
+/*
+ * Code taken from https://stackoverflow.com/questions/20743859/imageview-rounded-corners/23713740#23713740
```

Comment:
This isn't the javadoc pattern: please look in other files to match them. Having me correct it in review is time-consuming for both of us.